scheduling-manager/src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration status="INFO" packages="com.amazonaws.services.lambda.runtime.log4j2">
+<Appenders>
+  <Console name="Console" target="SYSTEM_OUT">
```

Comment:
before shipping: We probably don't want both a Console and a Lambda appender when running inside an actual lambda function, since that will duplicate every log message.